src/amo/components/Addon/index.js

```diff
@@ -295,7 +303,7 @@ export class AddonBase extends React.Component {
       descriptionProps.children = <LoadingText width={100} />;
     }
 
-    return (
+    return !hideAbout ? (
```

Comment:
Maybe we should invert the condition and use a variable like `showAboutCard`? It seems weird to negate this variable to follow the happy path.

Reply:
I actually had the but it seemed that I had to check for more things that way...

but I'll double ck this again..

Reply:
ah I see what you mean, then maybe invert the condition when you assign the value, but not here.